lib/spack/spack/solver/asp.py

```diff
@@ -1144,13 +1145,19 @@ def on_model(model):
                 solve_kwargs["on_unsat"] = cores.append
 
             timer.start("solve")
+            start_time = time.monotonic()
```

Comment:
This is an *immense* nitpick, but the documentation really emphasizes that the result of `time.monotonic()` is not really a "time" in any sense, and is only meaningful when calculating the duration between calls. Especially since it's also being moved up above the `spack.config.CONFIG.get()` calls, I'm worried that this subtle distinction might be lost if we call it `start_time` and we later forget that it's not actually a "time". Maybe just `start_point`?

This is really python's fault for poor naming (rust makes it slightly easier: https://doc.rust-lang.org/std/time/struct.Instant.html. Too bad mypy isn't very flexible....).

Please feel free to ignore this comment.